code/game/objects/items/weapons/twohanded.dm

```diff
@@ -1127,3 +1128,108 @@
 			return ..()
 
 #undef BROOM_PUSH_LIMIT
+
+/obj/item/twohanded/supermatter  //Supermatter Halberd, used by Oblivion Enforcers
```

Comment:
this typepath could be much better. we are phasing out the /twohanded/` paths IIRC so that will leave this as `/obj/item/supermatter` which is not very descriptive. `/obj/item/supermatter_halberd` would be better